src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -296,9 +296,12 @@ private function getContainerEnvVars(ContainerBuilder $container): array
             return [];
         }
 
-        $file = file_get_contents($container->getParameter('debug.container.dump'));
-        preg_match_all('{%env\(((?:\w++:)*+\w++)\)%}', $file, $envVars);
-        $envVars = array_unique($envVars[1]);
+        $envVarsFromContainer = [];
```

Comment:
Let's not rename variables to make review easier.

```suggestion
        $envVars = [];
```